docs/releases/1.10.txt

```diff
@@ -493,6 +497,29 @@ In older versions, assigning ``None`` to a non-nullable ``ForeignKey`` or
 ``OneToOneField`` raised ``ValueError('Cannot assign None: "model.field" does
 not allow null values.')``. For consistency with other model fields which don't
 have a similar check, this check is removed.
+Timezone-aware :class:`~django.core.files.storage.Storage` API
```

Comment:
Missing a blank above, likely because of `docs/releases/*.txt merge=union` in `.gitattributes`.